### ISSUE TITLE
Custom Main Menu is missing Japanese font

### ISSUE THREAD

**Issue by rgmine** [ImpactDevelopment/ImpactIssues#1749]
### **Describe the bug**
I'm Japanese so using japanese language in settings.
but menu font won't show.I think japanese font is missing.
### **To Reproduce**
1. Go to 'Impact Settings'
2. Click on 'Click GUI'
3. Right click on 'Main Menu' and click 'Custom'
4. See this screenshot
![Screenshot at 2019-10-14 10-48-14](https://user-images.githubusercontent.com/56525233/66726106-6504b300-ee72-11e9-8261-96ccbeed889c.png)

**Comment by theluigiduck** [comment]
Sorry, but Impact doesn't really support other languages.

**Comment by LeafHacker** [comment]
The custom font Impact uses doesn't cover all of unicode. I don't think CFont allows fallthrough to a different font.

Perhaps a ttf could be built that includes both a fallback font and the default font where available.

As a work around, try disabling `custom` main menu

**Comment by ZeroMemes** [comment]
Pretty sure there wouldn't be any issues rendering our custom buttons using minecraft's font renderer, might want to try that or at least add it as an option?

**Comment by rgmine** [comment]
Can I change fonts such as 'TTF Font' option in 'HUD' settings?

**Comment by ZeroMemes** [comment]
The TTF Font setting under HUD will not affect the buttons used on the main menu, so that won't fix the issue.

**Comment by rgmine** [comment]
OK, I will disable custom main menu. Thanks all!

**Comment by LeafHacker** [comment]
> Pretty sure there wouldn't be any issues rendering our custom buttons using minecraft's font renderer, might want to try that or at least add it as an option?

Yeah our widgets should work with the vanilla font, the only issue I see with this is it feels like fixing a bug by just _addimg a workaround_. Ideally the font would do a proper fallback if a unicode character isn't present in the primary font. But I'm probably being unrealistic expecting this from CFont lol, so I guess an option is the best case :/
